src/Symfony/Component/DependencyInjection/Attribute/Closure.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Attribute;
+
+#[\Attribute(\Attribute::TARGET_FUNCTION)]
+class Closure
+{
+    public function __construct(
+        public ?string $name = null,
```

Comment:
the argument should be mandatory (if we keep the class)

Reply:
This class is not present anymore.